src/web/pages/tasks/TaskListPage.tsx

```diff
@@ -35,6 +34,7 @@ interface TaskEntitiesPageProps {
   onContainerTaskCreateClick?: () => void;
   onModifyTaskWizardClick?: () => void;
   onNewAgentTaskClick?: () => void;
+  onNewContainerImageClick?: () => void;
```

Comment:
Maybe rename it to
```suggestion
  onNewContainerImageTaskClick?: () => void;
```
?